### DISCUSSION TITLE
Motrix v1.8.14

### DISCUSSION THREAD

**Discussion by agalwood** [agalwood/Motrix#1457] [Releases]
## What's Changed
* fix: app auto update failed by @agalwood in https://github.com/agalwood/Motrix/pull/1455

**Full Changelog**: https://github.com/agalwood/Motrix/compare/v1.8.12...v1.8.14

<hr /><em>This discussion was created from the release <a href='https://github.com/agalwood/Motrix/releases/tag/v1.8.14'>Motrix v1.8.14</a>.</em>

**Comment by whisper0425** [comment]
我用的mac系统的chrome 和 EDGE 都安装了 motrix webextension 也都按照要求配置了 程序 和 插件的 RPC授权密钥。软件也是打开的状态，插件也是“绿色“的开关按钮 但是下载任务始终无法连接到程序中 在插件中点击开关图标至 “红色” 时 弹出默认的浏览器下载 怎么解决这个问题呢。
MacBook pro A2141 macOS 13.3.1

**Reply by CW-B-W** [reply]
有沒有 Errors，如圖中發送錯誤時，該會出現 Errors 選項在 edge://extensions 頁面
<img width="912" alt="Screen Shot 2023-04-28 at 4 37 49 AM" src="https://user-images.githubusercontent.com/76680670/234985162-c25a2c34-8a36-4369-9db2-b97bcf43bb80.png">

**Comment by Bmswad1** [comment]
how to install windows appx version?
